examples/02_miscellaneous/plot_1_pearson_correlation.py

```diff
@@ -177,12 +189,10 @@ def correlation_callback(
 )
 
 # %%
-# We finally plot the evolution of the Pearson correlation and
-# the training loss at each BCD iteration. We can see that the
-# Pearson correlation dips at the first iteration, but quickly
-# increases and stabilizes at the second iteration. This
-# is due to the initialization of the transport plan, which
-# is filled with ones.
+# The Pearson correlation and training loss evolution are then plotted for
+# each BCD iteration. As we begin over-fitting the training data, the
+# correlation rises during the initial iterations before gradually
+# falling down.
```

Comment:
I don't think you can make such a claim using training data only.
In particular, I think you can only deduce that your algorithm is over-fitting by looking at metrics computed on validation data.
However, what we can say here is that a lower fugw loss does not always result in higher functional correlations.